### ISSUE TITLE
Tusk Chest Reward SUCKS

### ISSUE THREAD

**Issue by lenincambosa** [ValveSoftware/Dota2-Gameplay#5499]
### Description

Super excited to play when week 12 was unlocked, thinking this is the time I can gain more BP levels but all I had was disappointment 3 chests opened, and a total of 4,000 BP points were received. What fucking greedy person Gaben is, to be honest, I'm not even excited about playing to get the 4th chest.

**Comment by s1raz1** [comment]
yea , cleared a few of the bonus weeks chest and  1k bp is average, nothing to get excited not to mention that the coordinator lags or flat DCs just when you kick the bs chest as well

**Comment by gaia1991** [comment]
true. it's happen to me too all bonus chest only 1k bp point even tuskar kick more times. very disappointed.
